test/url_helpers_test.rb

```diff
@@ -106,7 +106,7 @@ class Spot < ModelBase
 end
 class SpotsController < InheritedResources::Base
   belongs_to :house do
-    belongs_to :dishwasher, singleton: true do
+    singleton_belongs_to :dishwasher do
```

Comment:
@javierjulio after a second look, I've noticed that singleton belongs to were actually tested but only through the `singleton: true` option.

Since `singleton: true` was passed in multiple specs, I've changed one to use the helper

Reply:
@tagliala that's awesome! Sounds good to me. Thanks again for seeing this through. Let me know once this is ready for review.